website/conferences/views.py

```diff
@@ -123,6 +127,19 @@ def add_poster_by_email(conference, message):
         presentation_type=message.conference_category.lower(),
         is_spam=message.is_spam,
     )
+    if node_created and user_created:
+        osfstorage = node.get_addon('osfstorage')
```

Comment:
Will this ever raise an exception? If so, let's wrap it. If not, let's just turn the `root_id` into a single line assignment as `osfstorage` isn't reused